src/components/textfield/textfield.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import styled from 'styled-components';
+import { TextfieldLabel } from './../typography/typography';
+
+const StyledLabel = styled(TextfieldLabel)`
+  display: block;
+  margin-bottom: 4px;
+
+  &.danger {
+    font-weight: 700;
+    color: #ff0d35;
+  }
+`;
+
+const Textfield = (props) => {
+  const StyledInputText = styled.input.attrs({
+    type: 'input',
+  })`
+    width: 320px;
+    height: 48px;
+
+    padding-left: 16px;
+    padding-right: 16px;
+
+    font-size: 16px;
+    line-height: 150%;
+    letter-spacing: -0.01em;
+    color: #202840;
+
+    border: 2px solid;
+    border-color: ${props.error ? '#FF0D35' : '#f7f8fa'};
+    background: #f7f8fa;
+  `;
+
+  return (
+    <label>
+      <StyledLabel>{props.label}</StyledLabel>
+      <StyledInputText />
+      <StyledLabel className="danger">
+        {props.error ? props.errorMessage : ''}
```

Comment:
Wild suggestion: we don't need an `error` property. We can simply assume that there is no error if the `errorMessage` is falsy (i.e. `"" (empty string)`, `null`, `undefined`, `0`, `false`, ...). If `errorMessage` is set to something truthy we can assume that this textfield has an error.